resolwe_bio/processes/reports/amplicon_report.yml

```diff
@@ -0,0 +1,49 @@
+# ===================
+# Generate PDF report
+# ===================
+
+- slug: amplicon-report
+  name: Amplicon report
+  requirements:
+    expression-engine: jinja
+  data_name: "Report ({{ bam|sample_name|default('?') }})"
+  version: 0.0.1
+  type: data:report:amplicon
+  category: analyses:variants
+  flow_collection: sample
```

Comment:
It is necessary, otherwise the created data object is not appended to the existing sample.